cmd/collector/app/sanitizer/zipkin/span_sanitizer.go

```diff
@@ -168,11 +168,12 @@ func (s *errorTagSanitizer) Sanitize(span *zc.Span) *zc.Span {
 		if binAnno.AnnotationType != zc.AnnotationType_BOOL && strings.EqualFold("error", binAnno.Key) {
 			binAnno.AnnotationType = zc.AnnotationType_BOOL
 
-			if strings.EqualFold("true", string(binAnno.Value)) || len(binAnno.Value) == 0 {
+			switch {
+			case strings.EqualFold("true", string(binAnno.Value)), len(binAnno.Value) == 0:
```

Comment:
I find this much harder to read (especially the fact that this is an OR

Reply:
Return if-else or replace `,` with a `||`?

Reply:
if you keep `||` then fine to do a switch. BTW, better to move `len(binAnno.Value) == 0` in the first position (left of `||`)

Reply:
Done in 20cd8d933f8eea211db522bfed1c4cf2a30e55a0.